### ISSUE TITLE
Mi 6 does not work normally after installation

### ISSUE THREAD

**Issue by qxf123369** [degdag/W.I.P_sagit_mainline#6]
Hello, I am a novice. After installing the image generated by the pmbootstrap tool to Mi 6 through the document, other functions are not normal except for ssh login.
I have been tossing for two weeks, can you help me generate an image that can run docker and wireguard.

**Comment by degdag** [comment]
It's been a while since I lost my mi6 so the documentation about it on postmarketos maybe too outdated for current Linux mainline 6.0 and up 
If you used my repo you should be able to get the same status as in postmarketos wiki page .

**Comment by degdag** [comment]
> Hello, I am a novice. After installing the image generated by the pmbootstrap tool to Mi 6 through the document, other functions are not normal except for ssh login. I have been tossing for two weeks, can you help me generate an image that can run docker and wireguard.

There is basically some build uploaded at xda if you want to test it.

**Comment by qxf123369** [comment]
> 

Can you send me the project link?

**Comment by qxf123369** [comment]
Why is there a black screen after flashing the kernel compiled in the document?

**Comment by qxf123369** [comment]
> It's been a while since I lost my mi6 so the documentation about it on postmarketos maybe too outdated for current Linux mainline 6.0 and up If you used my repo you should be able to get the same status as in postmarketos wiki page .

Why is there a black screen after flashing the kernel compiled in the document?

**Comment by degdag** [comment]
> > It's been a while since I lost my mi6 so the documentation about it on postmarketos maybe too outdated for current Linux mainline 6.0 and up If you used my repo you should be able to get the same status as in postmarketos wiki page .
> 
> Why is there a black screen after flashing the kernel compiled in the document?

Dunno why it should be working fine as I tested.
Sorry I can't help much I don't have the device anymore

**Comment by qxf123369** [comment]
> > > 我丢失我的 mi6 已经有一段时间了，所以 postmarketos 上关于它的文档对于当前的 Linux 主线 6.0 及更高版本来说可能太过时了如果你使用我的 repo，你应该能够获得与 postmarketos wiki 页面中相同的状态。
> > 
> > 
> > 刷入文档中编译的内核后为什么会黑屏？
> 
> 不知道为什么它在我测试时应该工作正常。 抱歉，我帮不上什么忙了，我没有设备了

 pmbootstrap init
 pmbootstrap install
 
 git clone https://github.com/degdag/W.I.P_sagit_mainline.git
 cd W.I.P_sagit_mainline/
 source ~/pmOS/pmbootstrap/helpers/envkernel.sh
 make sagit_defconfig
 make -j`nproc`
 
 cp /home/aa/sagit/chroot_rootfs_xiaomi-sagit/boot/boot.img ~/downstream-boot.img
 
 pmbootstrap chroot -- apk add abootimg android-tools mkbootimg dtbtool
 export DEVICE=`pmbootstrap config device | cut -d'-' -f2` WORK=`pmbootstrap config work` PMOS_BOOT_IMG_PATH="~/downstream-boot.img"
 export TEMP="$WORK/chroot_native/tmp/mainline/" DTB="$PWD/.output/arch/arm64/boot/dts/qcom/msm8998-$DEVICE.dtb"
 rm -r "$TEMP"; mkdir -p "$TEMP"
 cat .output/arch/arm64/boot/Image.gz "$DTB" > "$TEMP/Image.gz-dtb"
 cp "$PMOS_BOOT_IMG_PATH" "$TEMP/boot.img"
 pmbootstrap chroot -- abootimg -u /tmp/mainline/boot.img -k /tmp/mainline/Image.gz-dtb -c "bootsize=67108864"

================

Here are my steps, can you help me to see what is missing
Could it be because of the version of pmbootstrap that causes the black screen after flashing?

**Comment by degdag** [comment]
I don't think so you might just have some mistake on how to make a working boot img from pmbootstrap , also you can just use regular way that pmbootstrap make bootimg.
Anyway as I mentioned before I don't have this device anymore so no much help can be provided ATM.

**Comment by kuailexs** [comment]
@qxf123369 也许你可以找找QQ群。现在他没有用这款手机了。想帮忙也不好帮了。
